lib/relayWebhooks.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+var api = 'relay-webhooks';
+
+var toApiFormat = function(input) {
+  var model = {
+    match: {}
+  };
+
+  model.target = input.target;
+  model.match.domain = input.domain;
+
+  model.name = input.name;
+  model.auth_token = input.authToken;
+  model.match.protocol = input.protocol;
+
+  return model;
+};
+
+module.exports = function(client) {
+  var relayWebhooks = {
+    all: function(callback) {
+      var reqOpts = {
+        uri: api
+      };
+      client.get(reqOpts, callback);
+    },
+    find: function(webhookId, callback) {
```

Comment:
let's call this argument `relayWebhookId` to avoid confusion. We should call it relayWebhookId in the error messages too.